tilavarauspalvelu/api/palvelukartta/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
```

Comment:
Would these fit better in `/api/rest/` 🤔

Reply:
This is specifically for palvelukartta, not for meant general use.